README.md

```diff
@@ -33,60 +33,54 @@ No, Hammerspace can be used standalone simply to intercept [PostMessage API](htt
 - HammerSpace used on the iframe to receive communication from any application on the host side
 - To handle communication where there are multiple iframes (web components) and distinguish the communication from each instance
 
-Hammerspace consists of two components:
-1) Rebound
-2) Client
+## Components
+**Rebound**
 
-## Rebound
+Rebound receives PostMessage messages whether it's setup on the host or the client. Messages can be sent and received from either one.
 
-Rebound is the part of HammerSpace that receives the messages, regardless if it is setup on the host or child (iframe). Rebound then relays the
-message to the Client.
+**Client**
 
-## Client
-
-The Client receives the messages from Rebound and fires an assigned callback based on the name of the event and the payload data in the message.
-The Client can handle as many events as required which are each assigned a callback. The Client is the part of HammerSpace that is used to
-run specific logic according to events that are received from the host.
+The Client is the part of HammerSpace that is used to run specific logic according to events that are received from the host. It receives the messages from Rebound and fires an assigned callback based on the name of the event and the payload data in the message. The Client can handle as many events as required which are each assigned a callback.
 
 ## Usage
```

Comment:
this whole section is a little wonky with spacing and is not consistent with the rest of the readme